pkg/daemon/rpm-ostree.go

```diff
@@ -99,6 +100,18 @@ func (r *RpmOstreeClient) GetBootedDeployment() (*RpmOstreeDeployment, error) {
 	return nil, fmt.Errorf("not currently booted in a deployment")
 }
 
+// Start ensures the daemon is running; the DBus activation timeout
+// is shorter than the systemd timeout.  xref
+// https://bugzilla.redhat.com/show_bug.cgi?id=1888565#c3
```

Comment:
Just wanted to confirm, this solves referenced bug but it doesn't solve bug https://bugzilla.redhat.com/show_bug.cgi?id=1888565 where we are seeing `rpm-ostree cleanup -r` failing because of slow disk, right?